### PR TITLE
add DELETE route to revoke API tokens

### PR DESCRIPTION
**Pull request by Elhebert**
### What does it do?

It adds an endpoint to revoke (delete) an api token.

### Why is it needed?

This will allow users to revoke tokens that shouldn't be used anymore (ie. for security reason).

### How to test it?

The endpoint is supported by unit and e2e tests.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: packages/core/admin/server/services/__tests__/api-token.test.js [hunk 1]
```diff
@@ -153,4 +153,49 @@ describe('API Token', () => {
...
+      const res = await apiTokenService.revoke(token.id);
+
+      expect(mockedDelete).toHaveBeenCalledWith({
+        select: ['id', 'name', 'description', 'type'],
+        where: { id: token.id },
+      });
+      expect(res).toEqual(token);
+    });
+
+    test('It returns `null` if the resource does not exists', async () => {
```
Comment (petersg83) [inline packages/core/admin/server/services/__tests__/api-token.test.js:183]: exist*

File: packages/core/admin/server/tests/admin-api-token.test.e2e.js [hunk 2]
```diff
@@ -175,4 +175,29 @@ describe('Admin API Token CRUD (e2e)', () => {
...
+    expect(res.statusCode).toBe(200);
+    expect(res.body.data).toStrictEqual({
+      name: 'api-token_tests-name-with-spaces-at-the-end',
+      description: 'api-token_tests-description-with-spaces-at-the-end',
+      type: 'read-only',
+      id: 3,
+    });
+  });
+
+  test('8. Does not return an error if the ressource does not exists', async () => {
```
Comment (petersg83) [inline packages/core/admin/server/tests/admin-api-token.test.e2e.js:194]: exist*

**Review by petersg83 — CHANGES_REQUESTED** [review]
It's perfect for me! Only 2 typos

**Review by petersg83 — APPROVED** [review]

File: packages/core/admin/server/controllers/api-token.js [hunk 3]
```diff
@@ -41,4 +41,12 @@ module.exports = {
 
     ctx.send({ data: apiTokens });
   },
+
+  async revoke(ctx) {
```
Comment (alexandrebodin) [inline packages/core/admin/server/controllers/api-token.js:45]: Any reason to call this revoke ? I looks to me it doesn't convey the actual functionally.

**Pull request merged**